### PR TITLE
Minor metric doc fixes

### PR DESCRIPTION
**Pull request by cijothomas**
Fixes #.

## Changes

Please provide a brief description of the changes here.

For significant contributions please make sure you have completed the following items:

* [ ] Appropriate `CHANGELOG.md` updated for non-trivial changes
* [ ] Design discussion issue #
* [ ] Changes in public API reviewed

### CODE REVIEW

File: docs/metrics/customizing-the-sdk/README.md [hunk 1]
```diff
@@ -229,7 +229,6 @@ By default, the boundaries used for a Histogram are [`{ 0, 5, 10, 25, 50, 75, 10
 Views can be used to provide custom boundaries for a Histogram. The measurements
 are then aggregated using the custom boundaries provided instead of the the
 default boundaries. This requires the use of `ExplicitBucketHistogramConfiguration`.
-[Monday 08:36 PM] Reiley Yang
```
Comment (reyang) [inline docs/metrics/customizing-the-sdk/README.md:232]: 😆

Comment (cijothomas) [reply docs/metrics/customizing-the-sdk/README.md:232]: 🤣

Comment (alanwest) [reply docs/metrics/customizing-the-sdk/README.md:232]: 🐇

File: src/OpenTelemetry.Exporter.Console/README.md [hunk 2]
```diff
@@ -17,12 +17,15 @@ dotnet add package OpenTelemetry.Exporter.Console
 ```
 
 See the
```
Comment (utpilla) [inline src/OpenTelemetry.Exporter.Console/README.md:19]: Remove this.

File: src/OpenTelemetry.Exporter.Console/README.md [hunk 3]
```diff
@@ -17,12 +17,15 @@ dotnet add package OpenTelemetry.Exporter.Console
...
-[`TestConsoleExporter.cs`](../../examples/Console/TestConsoleExporter.cs) for an
-example of how to use the exporter for exporting traces.
-
 See the [Program](../../docs/logs/getting-started/Program.cs) for
 an example of how to use the exporter for exporting logs.
 
+See the [Program](../../docs/metrics/getting-started/Program.cs) for
```
Comment (reyang) [inline src/OpenTelemetry.Exporter.Console/README.md:23]: Consider changing these to bullet points:

See the individual "getting started" examples depending on the signal being used:
* Logs - [/docs/logs/getting-started/Program.cs](../../docs/logs/getting-started/Program.cs)
* Metrics - [/docs/metrics/getting-started/Program.cs](../../docs/metrics/getting-started/Program.cs)
* Traces - [/docs/traces/getting-started/Program.cs](../../docs/traces/getting-started/Program.cs)

**Review by reyang — APPROVED** [review]

**Review by alanwest — APPROVED** [review]

**Review by utpilla — APPROVED** [review]
Left a comment to remove an extra "See the".

**Review by reyang — APPROVED** [review]

**Pull request merged**